packages/lisk-transactions/src/utils/validation/validator.ts

```diff
@@ -12,9 +12,10 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
-import Ajv from 'ajv';
-import addKeywords from 'ajv-merge-patch';
-import BigNum from 'browserify-bignum';
+import * as Ajv from 'ajv';
+// tslint:disable-next-line no-require-imports
+import addKeywords = require('ajv-merge-patch');
```

Comment:
why do we need to use require here?

Reply:
when library is exposed only function like `module.exports = function (xx) {}`, it cannot `import *` and not as default either